### PR TITLE
Added a dice command with buttons

### PR DESCRIPTION
**Pull request by RopraMMC**
I want to cry

### CODE REVIEW

**Comment by RopraMMC** [comment]
@Pandabweer plsssssss accept

File: src/cogs/fun.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
+from ast import alias
 import asyncio
+from re import S
+from sqlite3 import IntegrityError
```
[the remark this answers is not in the document]
Comment (RopraMMC) [reply src/cogs/fun.py:5]: not mine, I think someone else added them and I father them through VSC

[the remark this answers is not in the document]
Comment (realstealthninja) [reply src/cogs/fun.py:5]: double check your commits with a lint innit

[the remark this answers is not in the document]
Comment (RopraMMC) [reply src/cogs/fun.py:5]: someone else imported it... not me :angy:

[the remark this answers is not in the document]
Comment (Dekriel) [reply src/cogs/fun.py:5]: who

[the remark this answers is not in the document]
Comment (RopraMMC) [reply src/cogs/fun.py:5]: idk

**Review by Yat-o — APPROVED** [review]
Nice

**Review by sifte — APPROVED** [review]

**Comment by Dekriel** [comment]
ok

**Pull request merged**